### ISSUE TITLE
Training schedule of the VGG16 with no BN.

### ISSUE THREAD

**Issue by Eric-mingjie** [pytorch/vision#571]
Hi. I am trying to train the vgg-16 model without batch normalization. 
I notice that the official model has a top1 accuracy above 70%. Can I know what is the training schedule of the vgg? For example, the learning rate, batch size and weight decay.

**Comment by Eric-mingjie** [comment]
@fmassa Hi! Thanks for the information.
